app/src/ui/app.tsx

```diff
@@ -348,6 +348,41 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.openCurrentRepositoryInExternalEditor()
       case 'select-all':
         return this.selectAll()
+      case 'show-release-notes-popup':
+        return this.props.dispatcher.showPopup({
+          type: PopupType.ReleaseNotes,
+          newRelease: {
+            latestVersion: '42.7.99',
+            datePublished: 'Awesomeber 71, 2025',
+            pretext: 'kas',
+            enhancements: [
+              {
+                kind: 'new',
+                message: 'An awesome new feature!',
```

Comment:
Can you define some `#id` and `@author-name` mentions in here so that we can test that linking works for issues and authors? The list on https://desktop.github.com/release-notes/ is a good reference if you want some existing data to work with.

Reply:
Good idea. Sprinkled in one of each. Let me know if you'd like to see more, or other variations of anything.